package.json

```
@@ -5,7 +5,7 @@
   "engines": {
     "node": "20.11.0"
   },
-  "description": "A history of the designs for the Find, Apply, Publish, Manage, Register and Support services",
+  "description": "A history of the designs for the Find, Apply, Publish, Manage, Register, Manage school placements, Claim funding for mentor training and Support services",
```

As per my forst comment. We should use an abbreviated version rather than the whole service name for 'Manage school placements' and 'Claim funding for mentor training.' This will follow the style used for the other services. For example:

```suggestion
  "description": "A history of the designs for the Find, Apply, Publish, Manage, Register, Placements, Claim and Support services",
```